test/components/submission/update-review-state.spec.js

```diff
@@ -73,6 +73,15 @@ describe('SubmissionUpdateReviewState', () => {
     modal.get('input[value="hasIssues"]').should.be.focused();
   });
 
+  it('does not require a comment in the text area', async () => {
+    testData.extendedSubmissions.createPast(1, { reviewState: 'hasIssues' });
+    const modal = mountComponent();
+    await modal.setProps({ state: true });
+    modal.getComponent(MarkdownTextarea).props().required.should.equal(false);
+    const required = await modal.get('textarea').attributes('required');
```

Comment:
I don't think you need `await` here, since there's no promise. And actually, I feel like as long as there's a test of the `required` prop among the `MarkdownTextarea` tests, there's no need to test the attribute in addition to the prop. (And like I said above, I actually think this test isn't strictly necessary as long as there's that `MarkdownTextarea` test.)

Reply:
i'm going to keep these tests of the `MarkdownTextarea` `required` prop in these two components (as a kind of... reminder that the behavior is that one is required and one is optional) but not have it actually check the textarea directly, just the prop.